client/src/pages/layouts/components/UserMenu.jsx

```diff
@@ -38,8 +51,38 @@ const UserMenu = (props) => {
           open={profileOpen}
           setAvatar={setAvatar}
           closeMenu={handleClose}
+          handleOpenSnackbar={handleOpenSnackbar}
         />
       )}
+      <Snackbar
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+        open={openSnackbar}
+        autoHideDuration={6000}
+        onClose={handleCloseSnackbar}
+        message="Profile Picture Updated"
+        action={
+          <>
+            <Button
+              color="secondary"
+              size="small"
+              onClick={handleCloseSnackbar}
+            >
+              UNDO
+            </Button>
+            <IconButton
```

Comment:
Hey Nick, Great Job!! Cool!!. I figured out two things
1. Once the profile photo submitted and updated, still upload dialog is open.
2. Can you clarify the purpose of the 'undo' button?

Reply:
Hi Swathi! Thanks for the input :)  
For 1) I fixed the issue so the menu closes, and 2) It was basically just another version of closing the Snackbar which is kind of unnecessary so I removed it too. Thanks for pointing that out.